database/tbcd/level/level.go

```diff
@@ -282,6 +283,27 @@ func (l *ldb) MetadataBatchGet(ctx context.Context, allOrNone bool, keys [][]byt
 	return l.transactionBatchGet(ctx, mdDB, allOrNone, keys)
 }
 
+func (l *ldb) BlockKeystoneByL2KeystoneAbrevHash(ctx context.Context, abrevhash chainhash.Hash) (*tbcd.Keystone, error) {
+	log.Tracef("BlockKeystoneByL2KeystoneAbrevHash")
+	defer log.Tracef("BlockKeystoneByL2KeystoneAbrevHash exit")
+
+	kssDB := l.pool[level.KeystonesDB]
```

Comment:
let's add a check here to ensure that this exists

```
kssDB, ok := ...
if !ok { ...
```

Reply:
If it doesn't exist we crash. That's intended because the caller did it wrong. This pattern is used all over the code.